script/package-version/package-version.inc.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Keyman\Site\com\keyman\api;
+
+require_once(__DIR__ . '/../../tools/util.php');
+
+class PackageVersion
+{
+  static function available_platforms()
+  {
+    return ['android', 'ios', 'linux', 'mac', 'web', 'windows'];
+  }
+
+  function execute($mssql, $params, $platform)
+  {
+    // TODO: params should be expanded to keyboards, models
```

Comment:
As in, made separate parameters?

Reply:
Yes. But I didn't want to make too many changes at present... There's far too much in flux as it is!

Reply:
Isn't that the truth!